runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -4066,6 +4081,39 @@ void amRequestShutdown(c_nodeid_t node) {
   amRequestCommon(node, &req, sizeof(req.b), false, NULL);
 }
 
+// amRequestFAMOResult
+//
+// This function is invoked by a fetching AMO handler to return the result
+// back to the initiating locale. The scenario is that locale A has sent a
+// fetching AMO to locale B via an AM. The AM handler on B invokes this
+// function to send an AM back to A that causes A to store the result of the
+// fetching AMO and set the pAmDone flag.
+
+static inline
+void amRequestFAMOResult(c_nodeid_t node,
```

Comment:
Note previous comments regarding `obj` vs `res` and not needing both `ofiType` and `size`.